### DISCUSSION TITLE
What are the advantages over RecyclerListView?

### DISCUSSION THREAD

**Discussion by swushi** [Shopify/flash-list#501] [Q&A]
I see you guys are using it internally. It seems that you guys are providing a simpler API then they did, but I am curious.. what else are you guys doing under the hood to make it perform better?

**Comment by Freddy03h** [comment]
I found 2 components with a native implementation in the source code : `AutoLayout` and `CellContainer` while RecyclerListView is still 100% JS.

Is it plan to backport those native components in RecyclerListView or is it too tied to FlashList?

Is FlashList has a plan to implement more advanced Layout? (idk if we can provide it a LayoutProvider built for RecyclerListView).
For example, a use case that it's seems not possible to implement with FlashList (because it's also not possible with Flat/SectionList), is a grid layout with section items.

Thank you!
